woocommerce/woocommerce
# Blocks E2E: Setup block theme without opening browser

### Changes proposed in this Pull Request:

In `block-theme.setup.ts`: 
- Verify the theme was set without opening the browser,
- Remove obsolete `wp rewrite flush` as it's already called within the preceding `wp rewrite structure` call,
- Add more assertions to ensure the env is setup correctly.

### How to test the changes in this Pull Request:

1. Change the theme slug in `block-theme.setup.ts` to a non-existing theme.
2. Run some test that uses the `block_theme.setup.ts`, for example: 

   ```
   pnpm test:e2e:side-effects -g "should show only products that match the filter"
   ```
   
3. Ensure the assertion failed and printed `Could not install and/or activate ${ BLOCK_THEME_SLUG }`.
